<html>
<head>
<title>TestNG:  Testedge</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>Testedge</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>19/7/0</td>
</tr><tr>
<td>Started on:</td><td>Thu Mar 06 11:10:03 CST 2025</td>
</tr>
<tr><td>Total time:</td><td>155 seconds (155501 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td><b>Attribute(s)</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='testNgTests.ArraysTest.clickTryHere1()'><b>clickTryHere1</b><br>Test class: testNgTests.ArraysTest</td>
<td><div><pre>java.lang.AssertionError: expected [true] but found [false]
	at testNgTests.ArraysTest.clickTryHere1(ArraysTest.java:90)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 32 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace691607801", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace691607801'><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1067)
	at org.testng.Assert.assertEquals(Assert.java:1031)
	at org.testng.Assert.assertEquals(Assert.java:1077)
	at testNgTests.ArraysTest.clickTryHere1(ArraysTest.java:90)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:739)
	at org.testng.TestRunner.run(TestRunner.java:614)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
	at org.testng.SuiteRunner.run(SuiteRunner.java:312)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>11</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
<tr>
<td title='testNgTests.ArraysTest.ArrayPracticeQuestion3()'><b>ArrayPracticeQuestion3</b><br>Test class: testNgTests.ArraysTest<br>Parameters: {Pythoncode=def search(lst, value):
    # Check if the value is in the list
    if value in lst:
        return "Element Found"
    return "Not Found"

# Example Usage
input_list = [12, 23, 45, 67, 6, 90]
value_to_search = 12
print(search(input_list, value_to_search))  # Output: Element Found

value_to_search = 25
print(search(input_list, value_to_search))  # Output: Not Found, output=Element Found
Not Found, AlertMessage=, SubmitionMessage=Submission Successful}</td>
<td><div><pre>java.lang.AssertionError: expected [] but found [SyntaxError: unindent does not match any outer indentation level on line 5]
	at testNgTests.ArraysTest.ArrayPracticeQuestion3(ArraysTest.java:280)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace647641020", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace647641020'><pre>java.lang.AssertionError: expected [] but found [SyntaxError: unindent does not match any outer indentation level on line 5]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:656)
	at org.testng.Assert.assertEquals(Assert.java:666)
	at testNgTests.ArraysTest.ArrayPracticeQuestion3(ArraysTest.java:280)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:739)
	at org.testng.TestRunner.run(TestRunner.java:614)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
	at org.testng.SuiteRunner.run(SuiteRunner.java:312)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>2</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
<tr>
<td title='testNgTests.ArraysTest.clickTryHere()'><b>clickTryHere</b><br>Test class: testNgTests.ArraysTest</td>
<td><div><pre>java.lang.AssertionError: expected [true] but found [false]
	at testNgTests.ArraysTest.clickTryHere(ArraysTest.java:48)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 32 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace2087524603", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace2087524603'><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1067)
	at org.testng.Assert.assertEquals(Assert.java:1031)
	at org.testng.Assert.assertEquals(Assert.java:1077)
	at testNgTests.ArraysTest.clickTryHere(ArraysTest.java:48)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:739)
	at org.testng.TestRunner.run(TestRunner.java:614)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
	at org.testng.SuiteRunner.run(SuiteRunner.java:312)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>11</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
<tr>
<td title='testNgTests.ArraysTest.ArrayPracticeQuestion4()'><b>ArrayPracticeQuestion4</b><br>Test class: testNgTests.ArraysTest<br>Parameters: {Pythoncode=def search(lst, value):
    # Check if the value is in the list
    if value in lst:
        return "Element Found"
    return "Not Found"

# Example Usage
input_list = [12, 23, 45, 67, 6, 90]
value_to_search = 12
print(search(input_list, value_to_search))  # Output: Element Found

value_to_search = 25
print(search(input_list, value_to_search))  # Output: Not Found, output=Element Found
Not Found, AlertMessage=, SubmitionMessage=Submission Successful}</td>
<td><div><pre>java.lang.AssertionError: expected [] but found [SyntaxError: unindent does not match any outer indentation level on line 5]
	at testNgTests.ArraysTest.ArrayPracticeQuestion4(ArraysTest.java:324)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace281656657", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace281656657'><pre>java.lang.AssertionError: expected [] but found [SyntaxError: unindent does not match any outer indentation level on line 5]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:656)
	at org.testng.Assert.assertEquals(Assert.java:666)
	at testNgTests.ArraysTest.ArrayPracticeQuestion4(ArraysTest.java:324)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:739)
	at org.testng.TestRunner.run(TestRunner.java:614)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
	at org.testng.SuiteRunner.run(SuiteRunner.java:312)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>1</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
<tr>
<td title='testNgTests.ArraysTest.ArrayPracticeQuestion2()'><b>ArrayPracticeQuestion2</b><br>Test class: testNgTests.ArraysTest<br>Parameters: {Pythoncode=def search(lst, value):
    # Check if the value is in the list
    if value in lst:
        return "Element Found"
    return "Not Found"

# Example Usage
input_list = [12, 23, 45, 67, 6, 90]
value_to_search = 12
print(search(input_list, value_to_search))  # Output: Element Found

value_to_search = 25
print(search(input_list, value_to_search))  # Output: Not Found, output=Element Found
Not Found, AlertMessage=, SubmitionMessage=Submission Successful}</td>
<td><div><pre>java.lang.AssertionError: expected [] but found [SyntaxError: unindent does not match any outer indentation level on line 5]
	at testNgTests.ArraysTest.ArrayPracticeQuestion2(ArraysTest.java:237)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1233554936", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1233554936'><pre>java.lang.AssertionError: expected [] but found [SyntaxError: unindent does not match any outer indentation level on line 5]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:656)
	at org.testng.Assert.assertEquals(Assert.java:666)
	at testNgTests.ArraysTest.ArrayPracticeQuestion2(ArraysTest.java:237)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:739)
	at org.testng.TestRunner.run(TestRunner.java:614)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
	at org.testng.SuiteRunner.run(SuiteRunner.java:312)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>1</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
<tr>
<td title='testNgTests.ArraysTest.ArrayPracticeQuestion1()'><b>ArrayPracticeQuestion1</b><br>Test class: testNgTests.ArraysTest<br>Parameters: {Pythoncode=def search(lst, value):
    # Check if the value is in the list
    if value in lst:
        return "Element Found"
    return "Not Found"

# Example Usage
input_list = [12, 23, 45, 67, 6, 90]
value_to_search = 12
print(search(input_list, value_to_search))  # Output: Element Found

value_to_search = 25
print(search(input_list, value_to_search))  # Output: Not Found, output=Element Found
Not Found, AlertMessage=, SubmitionMessage=Submission Successful}</td>
<td><div><pre>java.lang.AssertionError: expected [] but found [SyntaxError: unindent does not match any outer indentation level on line 5]
	at testNgTests.ArraysTest.ArrayPracticeQuestion1(ArraysTest.java:176)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace242495499", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace242495499'><pre>java.lang.AssertionError: expected [] but found [SyntaxError: unindent does not match any outer indentation level on line 5]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:656)
	at org.testng.Assert.assertEquals(Assert.java:666)
	at testNgTests.ArraysTest.ArrayPracticeQuestion1(ArraysTest.java:176)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:739)
	at org.testng.TestRunner.run(TestRunner.java:614)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
	at org.testng.SuiteRunner.run(SuiteRunner.java:312)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>1</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
<tr>
<td title='testNgTests.ArraysTest.ArraySubmittion()'><b>ArraySubmittion</b><br>Test class: testNgTests.ArraysTest<br>Parameters: {Pythoncode=def search(lst, value):
    # Check if the value is in the list
    if value in lst:
        return "Element Found"
    return "Not Found"

# Example Usage
input_list = [12, 23, 45, 67, 6, 90]
value_to_search = 12
print(search(input_list, value_to_search))  # Output: Element Found

value_to_search = 25
print(search(input_list, value_to_search))  # Output: Not Found, output=Element Found
Not Found, AlertMessage=, SubmitionMessage=Submission Successful}</td>
<td><div><pre>java.lang.AssertionError: expected [Submission Successful] but found [Error occurred during submission]
	at testNgTests.ArraysTest.ArraySubmittion(ArraysTest.java:206)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1903177023", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1903177023'><pre>java.lang.AssertionError: expected [Submission Successful] but found [Error occurred during submission]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:656)
	at org.testng.Assert.assertEquals(Assert.java:666)
	at testNgTests.ArraysTest.ArraySubmittion(ArraysTest.java:206)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:739)
	at org.testng.TestRunner.run(TestRunner.java:614)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
	at org.testng.SuiteRunner.run(SuiteRunner.java:312)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>2</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td><b>Attribute(s)</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='testNgTests.ArraysTest.clickApplicationsOfArray()'><b>clickApplicationsOfArray</b><br>Test class: testNgTests.ArraysTest</td>
<td></td>
<td>0</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
<tr>
<td title='testNgTests.ArraysTest.ArrayPracticeQuestion1()'><b>ArrayPracticeQuestion1</b><br>Test class: testNgTests.ArraysTest<br>Parameters: {Pythoncode=test, output=, AlertMessage=NameError: name 'test' is not defined on line 1, SubmitionMessage=Error occurred during submission}</td>
<td></td>
<td>1</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
<tr>
<td title='testNgTests.ArraysTest.clickArraysUsingList()'><b>clickArraysUsingList</b><br>Test class: testNgTests.ArraysTest</td>
<td></td>
<td>0</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
<tr>
<td title='testNgTests.ArraysTest.ArrayPracticeQuestion4()'><b>ArrayPracticeQuestion4</b><br>Test class: testNgTests.ArraysTest<br>Parameters: {Pythoncode=test, output=, AlertMessage=NameError: name 'test' is not defined on line 1, SubmitionMessage=Error occurred during submission}</td>
<td></td>
<td>2</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
<tr>
<td title='testNgTests.ArraysTest.testPythoncode1()'><b>testPythoncode1</b><br>Test class: testNgTests.ArraysTest<br>Parameters: {Pythoncode=print("hello"), ExpectedResult=hello, ExpectedAlert=}</td>
<td></td>
<td>11</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
<tr>
<td title='testNgTests.ArraysTest.clickBasicOperationsInLists()'><b>clickBasicOperationsInLists</b><br>Test class: testNgTests.ArraysTest</td>
<td></td>
<td>0</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
<tr>
<td title='testNgTests.ArraysTest.testPythoncode1()'><b>testPythoncode1</b><br>Test class: testNgTests.ArraysTest<br>Parameters: {Pythoncode=hello, ExpectedResult=, ExpectedAlert=NameError: name 'hello' is not defined on line 1}</td>
<td></td>
<td>0</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
<tr>
<td title='testNgTests.ArraysTest.clickSquaresofSortedArray()'><b>clickSquaresofSortedArray</b><br>Test class: testNgTests.ArraysTest</td>
<td></td>
<td>1</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
<tr>
<td title='testNgTests.ArraysTest.ArrayPracticeQuestion2()'><b>ArrayPracticeQuestion2</b><br>Test class: testNgTests.ArraysTest<br>Parameters: {Pythoncode=test, output=, AlertMessage=NameError: name 'test' is not defined on line 1, SubmitionMessage=Error occurred during submission}</td>
<td></td>
<td>1</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
<tr>
<td title='testNgTests.ArraysTest.testPythoncode()'><b>testPythoncode</b><br>Test class: testNgTests.ArraysTest<br>Parameters: {Pythoncode=hello, ExpectedResult=, ExpectedAlert=NameError: name 'hello' is not defined on line 1}</td>
<td></td>
<td>1</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
<tr>
<td title='testNgTests.ArraysTest.clickPracticeQuestions()'><b>clickPracticeQuestions</b><br>Test class: testNgTests.ArraysTest</td>
<td></td>
<td>0</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
<tr>
<td title='testNgTests.ArraysTest.clickMaxConsecutiveones()'><b>clickMaxConsecutiveones</b><br>Test class: testNgTests.ArraysTest</td>
<td></td>
<td>1</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
<tr>
<td title='testNgTests.ArraysTest.ArraySubmittion()'><b>ArraySubmittion</b><br>Test class: testNgTests.ArraysTest<br>Parameters: {Pythoncode=test, output=, AlertMessage=NameError: name 'test' is not defined on line 1, SubmitionMessage=Error occurred during submission}</td>
<td></td>
<td>3</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
<tr>
<td title='testNgTests.ArraysTest.clickArraysinPython()'><b>clickArraysinPython</b><br>Test class: testNgTests.ArraysTest</td>
<td></td>
<td>0</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
<tr>
<td title='testNgTests.ArraysTest.clickArray_Dropdown()'><b>clickArray_Dropdown</b><br>Test class: testNgTests.ArraysTest</td>
<td></td>
<td>0</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
<tr>
<td title='testNgTests.ArraysTest.clickFindNumswithEvenNumberOfDigits()'><b>clickFindNumswithEvenNumberOfDigits</b><br>Test class: testNgTests.ArraysTest</td>
<td></td>
<td>1</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
<tr>
<td title='testNgTests.ArraysTest.testPythoncode()'><b>testPythoncode</b><br>Test class: testNgTests.ArraysTest<br>Parameters: {Pythoncode=print("hello"), ExpectedResult=hello, ExpectedAlert=}</td>
<td></td>
<td>11</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
<tr>
<td title='testNgTests.ArraysTest.clickSearchTheArray()'><b>clickSearchTheArray</b><br>Test class: testNgTests.ArraysTest</td>
<td></td>
<td>1</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
<tr>
<td title='testNgTests.ArraysTest.ArrayPracticeQuestion3()'><b>ArrayPracticeQuestion3</b><br>Test class: testNgTests.ArraysTest<br>Parameters: {Pythoncode=test, output=, AlertMessage=NameError: name 'test' is not defined on line 1, SubmitionMessage=Error occurred during submission}</td>
<td></td>
<td>1</td>
<td>testNgTests.ArraysTest@6057aebb</td></tr>
</table><p>
</body>
</html>